adal/build.gradle

```diff
@@ -133,7 +134,7 @@ dependencies {
     snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '3.0.6', changing: true)
 
     // 'dist' flavor dependencies
-    distApi("com.microsoft.identity:common:3.0.6") {
+    distApi("com.microsoft.identity:common:3.4.5-RC2") {
```

Comment:
Are we using RC2 in the adal release?

Reply:
No, this was a mismerge based on an out-of-date local repo.